pyomo/contrib/doe/fim_doe_tutorial.ipynb

```diff
@@ -0,0 +1,2017 @@
+    }
+   ],
+   "source": [
+    "# Create model function\n",
+    "createmod = create_model\n",
+    "\n",
+    "# discretization by Pyomo.DAE\n",
+    "disc = disc_for_measure\n",
+    "\n",
+    "# Control time set [h]\n",
+    "t_control = [0, 0.125, 0.25, 0.375, 0.5, 0.625, 0.75, 0.875, 1]\n",
+    "    \n",
+    "# Measurement time points [h]\n",
+    "t_measure = [0, 0.125, 0.25, 0.375, 0.5, 0.625, 0.75, 0.875, 1]\n",
+    "\n",
+    "# design variable and its control time set\n",
+    "dv_pass = {'CA0': [0],'T': t_control}\n",
+    "    \n",
+    "# Create measurement object\n",
+    "measure_pass = {'C':{'CA': t_measure, 'CB': t_measure, 'CC': t_measure}}\n",
+    "measure_class =  Measurements(measure_pass)\n",
+    "\n",
+    "# Define parameter nominal value \n",
+    "parameter_dict = {'A1': 84.79085853498033, 'A2': 371.71773413976416, 'E1': 7.777032028026428, 'E2': 15.047135137500822}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "47137cb2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def generate_exp(t_set, CA0, T):  \n",
+    "    '''Generate experiments. \n",
+    "    t_set: time control set for T.\n",
+    "    CA0: CA0 value\n",
+    "    T: A list of T \n",
+    "    '''\n",
+    "    assert(len(t_set)==len(T)), 'T should have the same length as t_set'\n",
+    "    \n",
+    "    T_con_initial = {}\n",
+    "    for t, tim in enumerate(t_set):\n",
+    "        T_con_initial[tim] = T[t]\n",
+    "        \n",
+    "    dv_dict_overall = {'CA0': {0: CA0},'T': T_con_initial}\n",
+    "    return dv_dict_overall"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "written-tours",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The prior information FIM: [[0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0.]\n",
+      " [0. 0. 0. 0.]]\n",
+      "Prior Det: 0.0\n",
+      "Eigenvalue of the prior experiments FIM: [0. 0. 0. 0.]\n",
+      "Eigenvalue of the prior experiments FIM: [[1. 0. 0. 0.]\n",
+      " [0. 1. 0. 0.]\n",
+      " [0. 0. 1. 0.]\n",
+      " [0. 0. 0. 1.]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# empty prior\n",
+    "prior_all = np.zeros((4,4))\n",
+    "\n",
+    "\n",
+    "# add prior information\n",
+    "#prior_5_300 = pd.read_csv('fim_5_300_scale.csv')\n",
+    "#prior_5_300_500 = pd.read_csv('fim_5_300_500_scale.csv')\n",
+    "\n",
+    "#prior_all = prior_5_300\n",
+    "\n",
+    "\n",
+    "prior_pass=np.asarray(prior_all)\n",
+    "\n",
+    "#L_initials = np.linalg.cholesky(prior_pass)\n",
+    "#print(L_initials)\n",
+    "\n",
+    "print('The prior information FIM:', prior_pass)\n",
+    "print('Prior Det:', np.linalg.det(prior_pass))\n",
+    "print('Eigenvalue of the prior experiments FIM:', np.linalg.eigvals(prior_pass))\n",
+    "print('Eigenvalue of the prior experiments FIM:', np.linalg.eigh(prior_pass)[1])"
```

Comment:
These are printing different things but have the same description. Is this a copy/paste typo?